CHANGELOG.md

```diff
@@ -12,6 +12,58 @@ rely on parsing the output from multiple nodes should pass the `--log-dir` flag
 to `tsh ssh`, which will create a directory where the separated output of each node
 will be written.
 
+#### Container Images
+
+Teleport 15 contains several breaking changes to improve the default
+security and usability of container images.
+
+##### "Fat" container images are discontinued
```

Comment:
s/Fat/Heavy/ - in Mac land, fat means multi-architecture and we are going harder on that.

Reply:
Yeah -- I didn't love "Fat" either, but "distrofull" seemed worse.  Will update to heavy.

Reply:
fixed in https://github.com/gravitational/teleport/pull/35452/commits/8407fc98aa1005736ecef01fe60d1e47cf140dea